package.json

```diff
@@ -58,12 +58,14 @@
     "parse-json": "^2.2.0",
     "path-is-inside": "^1.0.2",
     "pkg-up": "^2.0.0",
+    "project-bin-path": "^0.0.2",
```

Comment:
This package is published https://yarnpkg.com/en/package/project-bin-path , it does the job of getting the path to bin, just thought this can be abstracted

Reply:
and source code is https://github.com/ajaymathur/project-bin-path